scripts/install2

```diff
@@ -291,6 +278,7 @@ if [ "$WITH_PROMETHEUS_SERVER" == true ]; then
     askModule "WITH_GRAFANA" "Grafana"
 fi
 askModule "WITH_EXTRA_TOOLS" "Non-Essential Tools Useful for Network Analysis"
+askModule "WITH_EXTRA_NETCAT" "Replacement netcat that supports IPv6"
```

Comment:
```suggestion
askModule "WITH_EXTRA_NETCAT" "Change netcat to a version that supports IPv6"
```

Reply:
Prompt prepends "INSTALL" in front of text.

`Install change netcat` does not read well